cmd/snap/cmd_abort.go

```diff
@@ -25,11 +25,7 @@ import (
 	"github.com/snapcore/snapd/i18n"
 )
 
-type cmdAbort struct {
-	Positional struct {
-		ID changeID
-	} `positional-args:"yes" required:"yes"`
-}
+type cmdAbort struct{ lastMixin }
```

Comment:
Nice refactor! I wonder if there is a more fitting name for `lastMixin` - it is also capturing the `changeID` so to me it was not obvious at first that lastMixin also has this job. Trouble is I don't have a good suggestion either: `lastChangeMixin`, `changeIDmixin` also sounds not brilliant.